src/animationLoop.js

```diff
@@ -86,7 +86,7 @@ export default function configAnimation(config = {}) {
 
   return function startAnimation(state, animationStep, animationRender) {
     for (let i = 0; i < animRunning.length; i++) {
-      let val = animRunning[i];
+      const val = animRunning[i];
```

Comment:
Sorry, maybe I should have written this down somewhere.
In general I'd like to use `const` to literally mean "it does not change, ever". No mutation if it's a collection. If there's a mutation then I can quickly peruse and spot it because it'd be a `let`. Helps my reasoning a lot.

So for the purpose of this lib I'd like to keep using `let` for anything mutable please.